### ISSUE TITLE
Keyboard Accessibility: Shift+Tab cannot escape searchable multiselects

### ISSUE THREAD

**Issue by jettoki** [monterail/vue-multiselect#481]
Normally when you Shift+Tab on a select field, the focus moves to the previous field along the tabindex. Right now, Shift+Tab can't seem to escape a searchable multiselect field, even after a value is selected.  The issue goes away if you mark the multiselect component as NOT searchable.

Repro: Check the first fiddle, open the second multiselect field and attempt to Shift+Tab.

**Searchable (not working)**
https://jsfiddle.net/7ma9xuvp/1/

**Unsearchable (working)**
https://jsfiddle.net/7ma9xuvp/2/

**Comment by justrhysism** [comment]
@jettoki is this different from #411?
